tensorflow_addons/options.py

```diff
@@ -0,0 +1,49 @@
+import os
+import platform
+import warnings
+import traceback
+
+try:
+    TF_ADDONS_PY_OPS = bool(int(os.environ["TF_ADDONS_PY_OPS"]))
+except KeyError:
+    if platform.system() == "Linux":
+        TF_ADDONS_PY_OPS = False
+    else:
+        TF_ADDONS_PY_OPS = True
```

Comment:
So MacOS and Windows should support CPU custom ops without issue. Similar to the question above about priority is this the way we want to handle this?

Reply:
https://github.com/tensorflow/addons/pull/913

I suspect that on CPU, pure python might be faster. I need to run some speed tests though.

Windows user expect ops to work on gpu because it's the most common use case. Since I don't know if there is any way to know if the op is going to be executed on gpu or cpu, I can't decide at the last moment if I can use pure Python or not.

If you have any suggestion on how to make it work on GPU on windows and use the custom C++ op when using the CPU, I'm open to ideas.

Reply:
Thoughts:

1) GPU Ops on Windows is an achievable task, just requires a bit of a time commitment: https://github.com/tensorflow/addons/issues/784
2) When you say pure python might be faster you mean with XLA right? I have a hunch the majority of users will not be applying that themselves, but at least they have the option to.
3) You make a good point regarding windows users not being able to select custom op instead of GPU. It is possible to do something like this with device placement but unweildy.

I'm good with macOS / Windows using python ops when they're available as default since it makes this quite a bit more concise. When windows GPU ops are added would we consider editing this option switch?